src/renderer/wg_engine/tvgWgGeometry.cpp

```diff
@@ -87,13 +87,12 @@ void WgPolyline::appendPoint(WgPoint pt)
 }
 
 
-void WgPolyline::appendCubic(WgPoint p1, WgPoint p2, WgPoint p3)
+void WgPolyline::appendCubic(WgPoint p1, WgPoint p2, WgPoint p3, size_t nsegs)
 {
     WgPoint p0 = pts.count > 0 ? pts.last() : WgPoint(0.0f, 0.0f);
```

Comment:
shouldn't this be handled as an error? starting from 0,0 most probably won't a correct, right?

Reply:
That`s a good idia. At least we can handle it as assert